packages/front-end/components/Settings/Team/MemberList.tsx

```diff
@@ -84,6 +87,7 @@ const MemberList: FC<{
               <tr key={id}>
                 <td>{member.name}</td>
                 <td>{member.email}</td>
+                <td>{member.dateCreated && datetime(member.dateCreated)}</td>
```

Comment:
I think we can figure out the member join date with 99.9% accuracy from existing data.  In the vast majority of cases, the user registers (i.e. creates a document in the `users` collection) immediately before accepting an invitation or creating an organization.

The `users` collection in Mongo doesn't have a `dateCreated` field unfortunately, but it does have the standard Mongo `_id` field, which contains a timestamp.

Given a `UserDocument` object, you can get a timestamp with:

```ts
user._id.getTimestamp()
```

There will be a handful of edge cases where the `UserDocument` date is way before their actual organization join date, but it's  going to be extremely rare.  I think it's better to show the wrong date for 0.1% of users than to show no date for 99.9% of users.

Reply:
I've made this change. 
<img width="1361" alt="image" src="https://user-images.githubusercontent.com/113377031/200038828-38d940a4-7293-4610-9a74-6ed10c06f3f0.png">